colin.spec

```diff
@@ -9,8 +9,8 @@
 %endif
 
 Name:           %{pypi_name}
-Version:        0.1.0
-Release:        1%{?dist}
+Version:        0.3.0
+Release:        0%{?dist}
```

Comment:
`Release` starts from `1`

Reply:
Thanks for the rpmdev-bumpspec!
Resolved.